elki/src/main/java/ch/ethz/globis/pht/util/PhIteratorBase.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2011-2014 ETH Zurich. All Rights Reserved.
+ *
+ * This software is the proprietary information of ETH Zurich.
+ * Use is subject to license terms.
+ */
+package ch.ethz.globis.pht.util;
+
+import java.util.Iterator;
+
+public interface PhIteratorBase<K, V, E> extends Iterator<V> {
```

Comment:
This is one of the things that comes from the full PhTree implementation. `Iterator` is not a perfect API, but it is well established and may help integrating the PH-Tree in other projects.